helm/node-health-chart/templates/psp.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: extensions/v1beta1
```

Comment:
I'm not certain if you need a specific psp either - it should pick up the default one. i didn't define one for net-exporter

Reply:
Not sure though. I am using a configmap and the `net-exporter` doesn't have this. I took the base here from the `kvm-operator` so I will sail with it I guess.

Reply:
If you're not accessing the configmap at runtime (e.g: via kubernetes api), then you shouldn't need any additional permission for it

Reply:
Dropped it.